dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/event/CsvEventService.java

```diff
@@ -202,6 +199,7 @@ public List<Event> read(InputStream inputStream, boolean skipFirst)
           dataValue.getProvidedElsewhere(),
           dataValue.getDataElement(),
           dataValue.getValue(),
+          dataValue.getGeometry(),
```

Comment:
Was this intentional?
I don't see this used inside the if construct unlike the remaining ones.